LTT/LTT_local.serv

```diff
@@ -0,0 +1,106 @@
+# Server commands to make Longturn Traditional Freeciv rules based on Civ2Civ3 ruleset
+#
+
+rulesetdir LTT
+
+# map
+set mapsize="PLAYER"
+set generator="FRACTAL"
+set barbarians=enabled  # Standard LTT games don't have Barbarians enabled, but good for local AI games
+set startpos="ALL"
+set huts=0
+set landmass=40
+set separatepoles=enabled
+set specials=300
+set steepness=40
+set temperature=70
+set tilesperplayer=500
+set topology="WRAPX|WRAPY|ISO"
+set wetness=40
+set alltemperate=disabled
+set tinyisles disabled
+#set revealmap=START  # Uncomment this for revealing the map for test games
+
+# general rules
+set nationset=all
+set citymindist=4
+set contactturns=1
+set dispersion=6
+set foggedborders=enabled
+set rapturedelay=3
+set restrictinfra=enabled
+set revolen=2
+set revolentype=FIXED
+set spacerace=enabled
+set startunits=cccccwwwwwwxx
+set techlevel=0
+set unreachableprotects=disabled
+set killunhomed=0
+set trading_tech=disabled
+set airliftingstyle="FROM_ALLIES|TO_ALLIES|DEST_UNLIMITED"
+set razechance=10
+set happyborders=enabled
+set occupychance=0
+set globalwarming=enabled
+
+
+# diplomacy, technology
+set diplchance=40
+set diplgoldcost=20
+set diplbulbcost=20
+set diplomacy=ALL
+
+set sciencebox=100
+set techlost_donor=0
+set techlost_recv=0
+set techpenalty=10
+set trademindist=999
+set trading_city=disabled
+set team_pooled_research=disabled
+set techlevel=0
+set techlossforgiveness -1
+set techlossrestore 50
+set freecost 50
+set diplcost 50
+set conquercost 50
+
+# technical
+set aifill=10
+set allowtake=""
+#cmdlevel info # Allows for hack level in local games
+set compresstype plain
+set ec_chat=enabled
+set ec_info=enabled 
+set ec_max_size=5000
+set ec_turns=9
+set maxplayers=126
+set saveturns=1
```

Comment:
Not really sure if this and scorelog is needed either. They pollute folders. Player can enable if needed before the game begins.